config/jobs/image-pushing/k8s-staging-kubebuilder.yaml

```diff
@@ -0,0 +1,23 @@
+postsubmits:
+
+  kubernetes-sigs/kubebuilder:
+    - name: post-kubebuilder-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-api-machinery-kubebuilder
+      decorate: true
+      branches:
+        - ^master$
+        - ^kube-rbac-proxy-releases$
+      spec:
+        serviceAccountName: gcb-builder
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/image-builder:v20230111-cd1b3caf9c
+            command:
+              - /run.sh
```

Comment:
It should build the proxy image right?

To build today we call: https://github.com/kubernetes-sigs/kubebuilder/blob/kube-rbac-proxy-releases/build/cloudbuild_kube-rbac-proxy.yaml

Which will call https://github.com/kubernetes-sigs/kubebuilder/blob/kube-rbac-proxy-releases/build/build.sh passing the tag version to do the job.

So, what this run.sh does?
What is the expected outcome when this PR get merged?

Reply:
I thought the run.sh will run initially which is present inside the image-builder.

Reply:
> To build today we call: https://github.com/kubernetes-sigs/kubebuilder/blob/kube-rbac-proxy-releases/build/cloudbuild_kube-rbac-proxy.yaml

To be non-disruptive, do we need to create another cloudbuild.yaml as here? Not entirely sure so keeping the PR as a draft (https://github.com/kubernetes-sigs/kubebuilder/pull/3246). cc: @camilamacedo86

Reply:
Hi @varshaprasad96,

> To be non-disruptive, do we need to create another cloudbuild.yaml as here? Not entirely sure so keeping the PR as a draft (https://github.com/kubernetes-sigs/kubebuilder/pull/3246). cc: @camilamacedo86

I do not think that we need to have other  cloudbuild.yaml
See that it is only a config file like gorelease one. We can call the same.

However, the new infra should build the images to the new registry
See that we inform the image registry in the build.sh: https://github.com/kubernetes-sigs/kubebuilder/blob/kube-rbac-proxy-releases/build/build.sh#L25

So, we need to change that to receive this info as an ARG OR just create another for the new infra.